mesa/visualization/solara_viz.py

```diff
@@ -449,7 +449,7 @@ def make_initial_grid_layout(layout_types):
         {
             "i": i,
             "w": 6,
-            "h": 10,
+            "h": 14,
```

Comment:
I tested. This is too tall for my screen (12.5"). The long term solution is probably too use `ResponsiveColumn`.

Reply:
There is a solution at https://github.com/widgetti/solara/discussions/158.